collections/_drafts/agent-forwarding.md

```diff
@@ -0,0 +1,276 @@
+---
+title: OpenINF Next-Gen Guidance on Agent Forwarding
+category: contributing
+authors: @OpenINF/wg-a-team
+publish: false
+---
+
+## Agent Forwarding
```

Comment:
```suggestion
```

there'd be no reason to have this here anymore since the document title would be included as part of the `deep-page` template